catalog-info.yaml

```diff
@@ -68,6 +68,8 @@ spec:
           access_level: BUILD_AND_READ
         cloud-tooling:
           access_level: BUILD_AND_READ
+        obs-ux-infra_services-team:
```

Comment:
@watson this is needed for the manual quality gate defined in `pipeline.tests-qa.yaml`. Is this one that targets `pipeline.kibana-tests.yaml` sufficient?

Reply:
Yes this should be sufficient to allow you to accept the manual gate in QA